src/components/macroview/rightPanel/editForms/MousePressForm.tsx

```diff
@@ -52,15 +72,20 @@ export default function MousePressForm({
     if (selectedElement.data.data.type !== 'DownUp') {
       return
     }
-    let duration
-    if (mousepressDuration === '') {
-      duration = 0;
-    } else {
+    let duration = DefaultMouseDelay
 
-      duration = parseInt(mousepressDuration)
-      if (Number.isNaN(duration)) {
-        return
-      }
+    if (mousepressDuration >= DefaultMouseDelay) {
+      duration = mousepressDuration
+    } else if (mousepressDuration < DefaultMouseDelay) {
+      toast({
+        title: 'Minimum duration',
+        description: 'Duration must be at least 20ms',
```

Comment:
You can insert the default value into the string

Reply:
Done: [4d6b91c](https://github.com/WootingKb/wooting-macros/pull/194/commits/4d6b91cc80b09d6838b997ddcc1b542b3abc7fb5)